### PR TITLE
Update main software repo README and clean markdown LINT errors from linked documents

### PR DESCRIPTION
**Pull request by FerralCoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,141 @@
...
+Reference to the latest stable Caliptra RTL (reference to its own repository).
+
+### [hw-model](/hw-model/)
 
 A high-level testing library for instantiating and manipulating models of the
 hardware. Intended to target multiple backends, including sw-emulator,
 verilator, and (hopefully) a future FPGA implementation.
 
-## [sw-emulator](/sw-emulator/)
+### [image](/image)
```
Comment (FerralCoder) [inline README.md:73]: I need help with a good description for this one

File: README.md [hunk 2]
```diff
@@ -1,44 +1,141 @@
...
+Caliptra runtime environment that is responsible for all Caliptra functionality
+after the boot process. This module is jumped to from FMC.
+
+### [sw-emulator](/sw-emulator/)
 
 Fast software-based simulation of the Caliptra hardware. This is the fastest and
 easiest way to test changes to firmware, but fidelity may not be perfect.
 
-## [drivers](/drivers/)
+### [systemrdl](/systemrdl)
```
Comment (FerralCoder) [inline README.md:109]: I need help with a good description for this one

File: README.md [hunk 3]
```diff
@@ -1,44 +1,141 @@
...
 
-A rust library containing drivers for the Caliptra hardware, intended to be
-used by firmware running on Caliptra's RISC-V cpu.
+.
+
+### [test](/test)
+
+Test suites for various portions of the Caliptra implementation.
+
+### [test-harness](/test-harness)
```
Comment (FerralCoder) [inline README.md:117]: I need help with a good description for this one

File: README.md [hunk 4]
```diff
@@ -1,44 +1,141 @@
...
+
+.
+
+### [ureg](/ureg)
```
Comment (FerralCoder) [inline README.md:121]: I need help with a good description for this one

**Comment by andreslagarcavilla** [comment]
@korran can you help with Lou's open questions?

@steph-morton can you take a quick look on any editorial issues? Thanks

**Review by steph-morton — COMMENTED** [review]
Some of the edit recommendations, especially in rom/dev/README.md, may be out of scope for this PR but would be good to make if possible during these linter updates.

File: rom/dev/README.md [hunk 5]
```diff
@@ -14,7 +14,6 @@
 | 04/27/2023 | 0.5.2   | Added Runtime SVN bit clarification                                      |
 | 08/15/2023 | 0.9     | Added LMS keys and signatures to image format                            |
 
-
 ## 2. Spec Opens
```
Comment (steph-morton) [inline rom/dev/README.md:18]: if in scope, I recommend deleting section numbers throughout ROM spec.

File: README.md (hunk 1 above)
Comment (jhand2) [reply README.md:73]: "Libraries for generating and verifying Caliptra firmware images"

File: README.md (hunk 3 above)
Comment (jhand2) [reply README.md:117]: "A library for building self-contained test firmware binaries. This set's up minimal infrastructure for panic-handling and provides macros for defining test cases in firmware"

Comment (steph-morton) [reply README.md:117]: `set's up`-->`sets up` and add period at the end.

File: README.md (hunk 1 above)
Comment (steph-morton) [reply README.md:73]: Recommend: Add a period at the end of the description.

**Review by jhand2 — DISMISSED** [review]

File: fmc/README.md [hunk 6]
```diff
@@ -1,20 +1,4 @@
...
-| 02/10/2023 | 0.2     | Incorporate feedback and decisions from Caliptra WG meetings                       |
-| 02/27/2023 | 0.4     | Update for decision that Anti-Rollback will be entirely handled by ROM             |
-|            |         | Add details/clarifications in FMC Boot steps including where to store artifacts    |
-|            |         | Add Firmware Handoff Table (FHT) definition                                        |
-| 03/21/2023 | 0.5     | Additional fields added to FHT                                                     |
-| 03/29/2023 | 0.5.1   | Changed the value for invalid address fields in FHT                                |
-| 06/27/2023 | 0.7     | Add state of data and key vaults before/after FMC execution                        |
-| 06/30/2023 | 0.7.1   | Simplify reset/update/recovery sections                                            |
-|            |         | Add more PCR handling details                                                      |
+# Caliptra - FMC Specification v1.0
```
Comment (andreslagarcavilla) [inline fmc/README.md:1]: Remove upon rebase to latest

**Review by steph-morton — APPROVED** [review]

**Comment by FerralCoder** [comment]
Branch is now rebased with latest from main

**Review by steph-morton — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -120,7 +120,7 @@ A library for building self-contained test firmware binaries. This sets up minim
 
 ### [ureg](/ureg)
 
-.
+Register abstraction and code generator to create register libraries. The `registers` directory has a binary that creates the register code including ureg.
```
Comment (steph-morton) [inline README.md:123]: unclear what `including ureg` means in context. Could:

- add a comma:`register code, including ureg.` 
- or reword: `that creates the register code that includes ureg.`

Comment (FerralCoder) [reply README.md:123]: thanks, updated

**Review by steph-morton — APPROVED** [review]

**Review by korran — APPROVED** [review]

**Review by rusty1968 — APPROVED** [review]

**Review by jhand2 — APPROVED** [review]

**Pull request merged**